glib/src/auto/functions.rs

```diff
@@ -861,3 +864,97 @@ pub fn uuid_string_is_valid(str: &str) -> bool {
 pub fn uuid_string_random() -> crate::GString {
     unsafe { from_glib_full(ffi::g_uuid_string_random()) }
 }
+
+#[cfg(windows)]
+#[cfg_attr(docsrs, doc(cfg(windows)))]
+#[doc(alias = "g_win32_check_windows_version")]
+pub fn win32_check_windows_version(
+    major: i32,
+    minor: i32,
+    spver: i32,
+    os_type: Win32OSType,
+) -> bool {
+    unsafe {
+        from_glib(ffi::g_win32_check_windows_version(
+            major,
+            minor,
+            spver,
+            os_type.into_glib(),
+        ))
+    }
+}
+
+#[cfg(windows)]
+#[cfg_attr(docsrs, doc(cfg(windows)))]
+#[doc(alias = "g_win32_error_message")]
+pub fn win32_error_message(error: i32) -> crate::GString {
+    unsafe { from_glib_full(ffi::g_win32_error_message(error)) }
+}
+
+#[cfg(windows)]
+#[cfg_attr(docsrs, doc(cfg(windows)))]
+#[doc(alias = "g_win32_ftruncate")]
+pub fn win32_ftruncate(f: i32, size: u32) -> i32 {
+    unsafe { ffi::g_win32_ftruncate(f, size) }
+}
+
+#[cfg(windows)]
+#[cfg_attr(docsrs, doc(cfg(windows)))]
+#[doc(alias = "g_win32_get_command_line")]
+pub fn win32_get_command_line() -> Vec<crate::GString> {
+    unsafe { FromGlibPtrContainer::from_glib_none(ffi::g_win32_get_command_line()) }
+}
+
+#[cfg(windows)]
+#[cfg_attr(docsrs, doc(cfg(windows)))]
+#[doc(alias = "g_win32_get_package_installation_directory")]
+pub fn win32_get_package_installation_directory(package: &str, dll_name: &str) -> crate::GString {
+    unsafe {
+        from_glib_full(ffi::g_win32_get_package_installation_directory(
+            package.to_glib_none().0,
+            dll_name.to_glib_none().0,
+        ))
+    }
+}
+
+#[cfg(windows)]
+#[cfg_attr(docsrs, doc(cfg(windows)))]
+#[doc(alias = "g_win32_get_package_installation_subdirectory")]
+pub fn win32_get_package_installation_subdirectory(
+    package: &str,
+    dll_name: &str,
+    subdir: &str,
+) -> crate::GString {
+    unsafe {
+        from_glib_full(ffi::g_win32_get_package_installation_subdirectory(
+            package.to_glib_none().0,
+            dll_name.to_glib_none().0,
+            subdir.to_glib_none().0,
+        ))
+    }
+}
+
+#[cfg(windows)]
+#[cfg_attr(docsrs, doc(cfg(windows)))]
+#[doc(alias = "g_win32_get_windows_version")]
+pub fn win32_get_windows_version() -> u32 {
+    unsafe { ffi::g_win32_get_windows_version() }
+}
+
+#[cfg(windows)]
+#[cfg_attr(docsrs, doc(cfg(windows)))]
+#[doc(alias = "g_win32_getlocale")]
+pub fn win32_getlocale() -> crate::GString {
+    unsafe { from_glib_full(ffi::g_win32_getlocale()) }
+}
+
+#[cfg(windows)]
+#[cfg_attr(docsrs, doc(cfg(windows)))]
+#[doc(alias = "g_win32_locale_filename_from_utf8")]
+pub fn win32_locale_filename_from_utf8(utf8filename: &str) -> crate::GString {
```

Comment:
This is wrong. The return string is not UTF-8 but in the local encoding

Reply:
Marked all those functions as ignored for now, whoever needs them could go through the process of fixing things later.